front/app/containers/App/index.tsx

```diff
@@ -239,6 +239,26 @@ class App extends PureComponent<Props, State> {
               },
             });
           });
+        } else if (
+          tenant.data.attributes.style &&
+          tenant.data.attributes.style.customFontURL
+        ) {
+          import('webfontloader').then((WebfontLoader) => {
+            const fontName = (
```

Comment:
Absolutely optional code style comment : you probably do something like :
`const { customFontName, cutomFontUrl } =  tenant.data.attributes.style as IAppConfigurationStyle;`